src/server/transaction.cc

```diff
@@ -577,7 +577,7 @@ void Transaction::PrepareMultiForScheduleSingleHop(Namespace* ns, ShardId sid, D
 }
 
 // Runs in the dbslice thread. Returns true if the transaction concluded.
-bool Transaction::RunInShard(EngineShard* shard, bool txq_ooo) {
+bool Transaction::RunInShard(EngineShard* shard) {
```

Comment:
removing `txq_ooo` and use `sd.local_mask` for this (historically OOO was determined by the coordinator but now we do it per shard)